x-pack/plugins/cases/server/authorization/index.ts

```diff
@@ -73,101 +73,111 @@ export const ECS_OUTCOMES: Record<string, EcsEventOutcome> = {
   unknown: 'unknown',
 };
 
+/**
+ * Determines if the passed in operation was a write operation.
+ *
+ * @param operation an OperationDetails object describing the operation that occurred
+ * @returns true if the passed in operation was a write operation
+ */
+export function isWriteOperation(operation: OperationDetails): boolean {
```

Comment:
nit: `export function isWriteOperation(operation: OperationDetails): operation is WriteOperations`

Reply:
hmm when I try to make it an official type guard I get this error

![image](https://user-images.githubusercontent.com/56361221/120530325-62d6cb80-c3ab-11eb-8510-8ca23eb986f6.png)

I think it's because the parameter isn't `operation: WriteOperations | ReadOperations`

Reply:
Yes, I think because if there is no relationship between the value's type and the type in the type predicate, the guard do not make sense. A solution would be to change the argument to `string`. I see that you don't use it as a typeguard but as a helper function so it is ok as it is :)